### PR TITLE
fix: fix issue on applehv when mounting volume (#130)

### PR DESCRIPTION
**Pull request by lstocchi**
This patch adds the :z bind option when mounting the volume to fix the issue on applehv.

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by lstocchi** [comment]
@jeffmaury have you run it on your MAC? Bc it does not work for @feloy

**Comment by lstocchi** [comment]
Closing for the moment as this does not work. I had a dirty env apparently. If you execute the mount using the cli it update volume permissions and then it works fine with desktop. 
By downloading the model again using desktop it does not work

**Comment by feloy** [comment]
I'm reopening to track the issue.

context: 
- mac M3
- appleHV 

As far as I can understand, mounting the file with the `:z` flag using the podman API is not making the necessary changes on the files to mount. 

The steps to investigate:

- start the container with AI Studio (with podman API), and :z => no access to file
- start the container with CLI and :z => access to file
- start again the container with AI Studio (with podman API), and :z (and the same file) => access to file

Details on the file:
[mount.md](https://github.com/projectatomic/studio-extension/files/14053871/mount.md)

File: packages/backend/src/managers/applicationManager.ts [hunk 1]
```diff
@@ -137,6 +137,9 @@ export class ApplicationManager {
                 Target: `/${modelName}`,
                 Source: modelPath,
                 Type: 'bind',
+                BindOptions: {
+                  Propagation: 'z',
```
Comment (jeffmaury) [inline packages/backend/src/managers/applicationManager.ts:141]: That is not compatible with the Podman Desktop API

**Comment by lstocchi** [comment]
Closing as it should work by using the updated podman-desktop api -> https://github.com/projectatomic/ai-studio/pull/231

**Pull request closed without merging**